client/client/app/components/ngbBlastSearchPanel/ngbBlastWholeGenomeView/ngbBlastWholeGenomeView.helper.js

```diff
@@ -0,0 +1,17 @@
+function getMaxChromosomeSize(chrArray) {
+    return chrArray.reduce((max, chr) => {
+        return Math.max(chr.size, max);
+    }, 0);
+}
+
+export function helper($mdDialog, projectContext) {
+
+  const blastResults = projectContext.chromosomes.slice(0,8);
+    return function($scope) {
+        $scope.chromosomes = blastResults;
+        $scope.maxChrSize = getMaxChromosomeSize(blastResults);
```

Comment:
`maxChrSize` is redundant here: we don't use it, just passing to the `<ngb-blast-whole-genome-view>`, so we can move the computation inside `ngbBlastWholeGenomeView.controller`